code/game/objects/items/toys/toys.dm

```diff
@@ -504,3 +504,30 @@
 							/obj/item/toy/prize/odysseus					= 1,
 							/obj/item/toy/prize/phazon						= 1
 							)
+
+/obj/item/toy/festivizer
+	name = "\improper C92 pattern 'Festivizer' decorator"
+	desc = "State of the art, WY-brand, high tech... ah who are we kidding, it's just a festivizer. You spot a label on it that says: <i> Attention: This device does not cover item in festive wire, but rather paints it a festive color. </i> What a rip!"
+	icon = 'icons/obj/items/marine-items_christmas.dmi'
+	icon_state = "barbed_wire"
+
+/obj/item/toy/festivizer/examine(mob/user)
+	. = ..()
+	to_chat(user, SPAN_BOLDNOTICE("You see another label on \the [src] that says: <i> INCLUDES SUPPORT FOR FOREIGN BIOFORMS! </i> You're not sure you like the sound of that.."))
+
+/obj/item/toy/festivizer/afterattack(atom/target, mob/user, proximity_flag, click_parameters)
+	. = ..()
+	if(target.color)
+		to_chat(user, SPAN_NOTICE("Tthe [target] is already colored, don't be greedy!"))
+		return
+	if(!target.Adjacent(user))
+		return
+	target.color = (pick("red", "green"))
+	target.visible_message(SPAN_GREEN("\The [target] has been festivized by [user]! Merry christmas!"))
+	to_chat(user, SPAN_GREEN("You festivize \the [target]! Merry christmas!"))
```

Comment:
```suggestion
	to_chat(user, SPAN_GREEN("You festivize \the [target]! Merry Christmas!"))
```

Reply:
u didn't do this carlarc

Reply:
i did i just didn't use the suggestion... i did it manually.... stfu!